core/include/detray/core/detector.hpp

```diff
@@ -48,29 +52,65 @@ template <template <typename, unsigned int> class array_type = darray,
           template <typename...> class vector_type = dvector,
           template <typename...> class jagged_vector_type = djagged_vector,
           typename alignable_store = static_transform_store<vector_type>,
```

Comment:
Unless it is necessary, let's untemplate `alignable_store` as well

Reply:
Can't say I'm a fan, but I assume it helps with the data view?

Reply:
Yes it does

Reply:
I suspected :) Its already done